src/components/HomePage.js

```diff
@@ -0,0 +1,50 @@
+import { useState } from 'react';
+import { useSelector } from 'react-redux';
+import DataRender from './DataRender';
+
+const HomePage = () => {
+  const countries = useSelector((state) => state);
+  const [searchItem, setSearchItem] = useState('');
+  const loadingStatus = countries.loading;
+  return (
+    <div>
+      <div className="nav_bar">
+        <span className="nav_header">Top 20 African countries COVID cases</span>
+        <i className="fa fa-microphone microphone_button" aria-hidden="true" />
+        <i className="fa fa-gear nav_buttons" aria-hidden="true" />
+      </div>
+      <div className="image_wrapper">
+        <div className="image_container" />
+      </div>
+      <p className="search_item_container">
+        Search:&nbsp;&nbsp;
+        <input
+          value={searchItem}
+          type="text"
+          onChange={(e) => setSearchItem(e.target.value)}
+          placeholder="Search by country"
+        />
+      </p>
+      <div className="country_link_container">
+        {loadingStatus === false
+          ? countries.data
+            .filter(
```

Comment:
- [ ] Your filter works as expected :+1: , however, we are asked to use a `Filter` stateless component to filter data retrieved from the API. So you will require a `Filter` component to achieve this.

Reply:
Hello @FrederickMih ,

I used filter there to filter the search item. I have used a stateless filter in my details page to filter the data fetched from the API.

Reply:
Hi @Micky373. You needs a component for that